frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -852,6 +862,10 @@
                         vm.submissionVisibility[details.results[i].id] = details.results[i].is_public;
                     }
 
+                    for (var i = 0; i < details.results.length; i++) {
+                        vm.baselineStatus[details.results[i].id] = details.results[i].baseline_submission;
```

Comment:
Why are we using another loop? I think we can do it in the same loop in line 861-863. Am I missing something?